cypress/integration/landing_page_spec.js

```diff
@@ -1,5 +1,83 @@
-describe('Shared Milk user flows', () => {
-  it('Should confirm that true is equal to true', () => {
-    expect(true).to.equal(true)
+describe('Shared Milk landing page user flow', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000')
   });
-});
\ No newline at end of file
+
+  it('should display page title and navigation links on page load', () => {
+    cy.get('h1').contains('No Use Cryin\' Over Shared Milk')
+    cy.get('.nav').contains('Home')
+    cy.get('.nav').contains('Need Milk')
+    cy.get('.nav').contains('Donate')
+    cy.get('.nav').contains('About Us')
+    cy.get('.nav').contains('Support')
+  });
+
+  it('should be able to navigate to the Need Milk page view', () => {
+    // cy.intercept(fetch call)
+    cy.get('.nav').contains('Need Milk').click()
+    // cy.wait(['api call']);
```

Comment:
@cagallo I've never don't wait's like this before. Teach me! :)

Reply:
Yes, it's super cool when you need to make sure you're awaiting your api call, not sure how it will work with graphQL!